zzk/service/state.go

```diff
@@ -577,8 +635,24 @@ func DeleteState(conn client.Connection, req StateRequest) error {
 		logger.Debug("No state to delete on host")
 	}
 
-	// Delete the service instance
 	sspth := path.Join(basepth, "/services", req.ServiceID, req.StateID())
+
+	// Delete the current state (status)
```

Comment:
In that case, should we return an error (if the node is missing) or just continue and not include it in the transaction?